packages/core/resources/icons/icons.json

```diff
@@ -2735,5 +2735,26 @@
     "tags": "circles, combine, cross",
     "group": "action",
     "content": "\\e765"
+  },
+  {
+    "name": "Blocked Person",
+    "className": "pt-icon-blocked-person",
+    "tags": "person, human, male, female, character, customer, individual, social, banned, prohibited",
+    "group": "interface",
+    "content": "\\e768"
+  },
+  {
+    "name": "Slash",
+    "className": "pt-icon-slash",
+    "tags": "divide, separate",
+    "group": "action",
+    "content": "\\e769"
+  },
+  {
+    "name": "Percentage",
+    "className": "pt-icon-percentage",
+    "tags": "modulo, modulus",
+    "group": "action",
```

Comment:
i'm not sure "action" is the best group for these three icons, but they're also not the only miscategorized one so not too fussed about this. one day we should clean up all this metadata.